packages/xo-web/src/common/intl/messages.js

```diff
@@ -11,7 +11,10 @@ const messages = {
   expiration: 'Expiration',
   hostIp: 'Host IP',
   keyValue: '{key}: {value}',
-  sslCertificate: 'SSL certificate',
+  esxiImportSslCertificate: 'Source has a valid SSL certificate',
+  esxiImportThin:
+    "Create a thin disk, this will require to read the data twice, and won't show in task or the first phase",
```

Comment:
"Thin mode: smaller disk but initial read phase will be invisible." or something like this (smaller and less detailed)